README.md

```diff
@@ -0,0 +1,9 @@
+# Zapytania-do-aplikacji-QA-Academy-Server
+
+## Wymagania
+* newman
```

Comment:
nie

Reply:
> nie

Wiem, ze nie jest wymagany, bo jest zawarty w projekcie, ale wydawało mi się, że skoro go używamy to warto nakreślić.